models/contacts.js

```diff
@@ -1,19 +1,73 @@
-// const fs = require('fs/promises')
+const fs = require("fs");
+const path = require("node:path");
+const { v1: uuidv1 } = require("uuid");
 
-const listContacts = async () => {}
+const pathContacts = path.join(__dirname, "contacts.json");
+console.log(pathContacts);
 
-const getContactById = async (contactId) => {}
+const listContacts = async () => {
+	try {
+		return fs.readFileSync(pathContacts);
+	} catch (err) {
+		console.log(err.message);
+	}
+};
 
-const removeContact = async (contactId) => {}
+const getContactById = (contactId) => {
+	try {
+		const data = JSON.parse(fs.readFileSync(pathContacts));
+		const oneContact = data.find((item) => item.id === contactId);
+		return oneContact;
+	} catch (err) {
+		console.log(err);
+	}
+};
 
-const addContact = async (body) => {}
+const removeContact = (contactId) => {
+	try {
+		const data = JSON.parse(fs.readFileSync(pathContacts));
+		const deletedContact = data.filter((item) => item.id === contactId);
+		return deletedContact;
+	} catch (err) {
+		console.log(err);
+	}
+};
 
-const updateContact = async (contactId, body) => {}
+const addContact = (body) => {
```

Comment:
For `add, remove and update`  methods you forgot to update json file with current data.